scripts/generate.js

```diff
@@ -0,0 +1,53 @@
+import * as fs from 'fs'
+import * as path from 'path'
+import { compileFromFile } from 'json-schema-to-typescript'
+import prettier from 'prettier'
+import { printLog, logAndExit } from './utils.js'
+
+const SCHEMAS_PATH = 'schemas'
+const PRETTIER_CONFIG = '.prettierrc.yml'
+
+const DEFINITIONS = [
+  { source: 'rum-events-schema.json', name: 'rum' },
+  { source: 'telemetry-events-schema.json', name: 'telemetry' },
+  {
+    source: 'session-replay-browser-schema.json',
+    name: 'browserSessionReplay',
+    options: { additionalProperties: false },
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏we should have put `additionalProperties` in the `schema` files, so it is enforced for everyone

Reply:
I thought you guys were happy with them for both RUM & Telemetry events?
If it's not the case and you can ensure it all works fine, we could set this to `false` for all generated definitions. It's specific to the `json-schema-to-typescript` dependency, not to JSON-schema.

Reply:
> I thought you guys were happy with them for both RUM & Telemetry events?

We are happy with additional properties on RUM (and Telemetry? I guess we didn't think much about it in this case) 🙂 

> If it's not the case and you can ensure it all works fine, we could set this to `false` for all generated definitions. It's specific to the `json-schema-to-typescript` dependency, not to JSON-schema.

In fact, it is [a JSON schema "keyword"](https://json-schema.org/understanding-json-schema/reference/object.html#additional-properties). We could add it to JSON-schema files so other teams could benefit from it if their JSON-schema library support it. One disadvantage is that it might be cumbersome to use it on every `object`...

Reply:
> In fact, it is [a JSON schema "keyword"](https://json-schema.org/understanding-json-schema/reference/object.html#additional-properties). We could add it to JSON-schema files so other teams could benefit from it if their JSON-schema library support it. One disadvantage is that it might be cumbersome to use it on every object...

Of course, that's the key we're using in [Session Replay's Attributes](https://github.com/DataDog/rum-events-format/blob/9e0f8b2ab26ce0e33b058d7d5d3663ab043847f5/schemas/session-replay/browser/attributes-schema.json?q=additionalProperties#L7) to declare the types for the `additionalProperties`.
Well because it'd be cumbersome to have to add it as `false` on every object, that's why I assumed you were meaning to define the _default_ value for it (what I did for the SessionReplay definitions generation).

Up to you. Could be a follow-up change as well.